### DISCUSSION TITLE
In a Nuxt 3 app, how can I modify the handler produced by the "aws-lambda" preset (.output/server/index.handler)?

### DISCUSSION THREAD

**Discussion by beunerd** [nitrojs/nitro#2309] [Q&A]
In my Nuxt 3 app's `nuxt.config.ts` I have nitro.preset as "aws-lambda". When running `nuxt build` this produces the handler at `.output/server/index.handler` which I've added to my `serverless.yml` file per the [directions](https://nitro.unjs.io/deploy/providers/aws).

I would like, however, to modify the handler so that I can incorporate ["serverless-plugin-warmup" as described](https://www.serverless.com/plugins/serverless-plugin-warmup#javascript).

Seems like I just need to set up a different handler (and point my `serverless.yml` at that one instead) that passes the event and context off to the one generated by nitro, something like:

```
export const myHandler = (event: any, context: any) => {
  import("./.output/server/index.mjs").then(({ handler }) => handler(event, context));
};
```

If that's the case, then where would I create this file?

And if this is not the case, is there another way to modify the handler produced by the "aws-lambda" nitro preset?

**Comment by dimasxp** [comment]
the same question how to properly way to modify the handler produced by the "aws-lambda" preset.
to combine this 2 cases:
1) if this direct call from s3 trigger to lambda. how to get event payload in index route ? 
await readBody(event) - is empty here
2) +default behavior for other paths
